packages/react-components/src/components/VideoTile.tsx

```diff
@@ -282,7 +282,7 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
 
   const placeholderOptions = {
     userId,
-    text: initialsName || displayName,
+    text: initialsName !== undefined ? initialsName : displayName,
```

Comment:
tiny nit: this can be written as 
```suggestion
    text: initialsName ?? displayName
```

Reply:
Nice, small tip I can repeat here for myself to remember it better 
The difference from || and ?? for string
 1. '' ?? displayName = ''
 2. '' || displayName = displayName